### PR TITLE
Fixes orbiting click catcher.

### PR DESCRIPTION
**Pull request by AnturK**
I'm open to any ideas how to recognize UI elements better.

### CODE REVIEW

File: code/_onclick/hud/screen_objects.dm [hunk 1]
```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius = 10, clockwise = FALSE, rotation_speed = 20, rotation_segments = 36, pre_rotation = TRUE, lockinorbit = FALSE)
```
Comment (coiax) [inline code/_onclick/hud/screen_objects.dm]: I think you don't need to specify the arguments if you're just going to ignore them.

File: code/_onclick/hud/screen_objects.dm [hunk 2]
```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius, clockwise, rotation_speed, rotation_segments, pre_rotation, lockinorbit)
```
Comment (RemieRichards) [inline code/_onclick/hud/screen_objects.dm]: uh, no, they meant ```/orbit()```, you don't need to pass in the args at all if you're just ignoring them.

Comment (AnturK) [inline code/_onclick/hud/screen_objects.dm]: I'd rather have signatures same in case some crazy person decides to call them through named args.

Comment (RemieRichards) [inline code/_onclick/hud/screen_objects.dm]: if you're having the signatures, put the default args back, if you're not putting the defaults back, remove the signatures.

your call, pick one.

Comment (AnturK) [inline code/_onclick/hud/screen_objects.dm]: Removing is easier from web interace 📯

**Comment by RemieRichards** [comment]
actually commit-message-antur it'd be your fault for justifying it via your laziness and the web editor :P

**Comment by AnturK** [comment]
Shh

**Pull request merged**